### PR TITLE
Feat:"added landing page"

### PR DESCRIPTION
**Pull request by sadathshariff**
Finished landing page
landing page

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+main {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+}
+.item {
+  padding: 1rem;
```
Comment (ImrozNezami) [inline style.css:6]: You can use utility class here, else everything looks good.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  text-align: center;
+}
+.hero-text {
+  font-size: 1.5rem;
+}
+.btns-div {
+  margin-top: 3rem;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
```
Comment (vivekmehto) [inline style.css]: the space between buttons is much larger than expected I think we can give some width to the container to fix this

File: style.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+.links li {
+  padding: 0 1rem;
+  font-weight: 500;
+  font-size: 1.5rem;
+}
+.heartColor {
+  color: red;
+}
+@media screen and (max-width: 800px) {
+  main {
```
Comment (vivekmehto) [inline style.css:52]: not sure about this. just a suggestion check the media query.

**Review by vivekmehto — COMMENTED** [review]
Overall code looks clean. just one suggestion for the buttons.

File: style.css (hunk 1 above)
Comment (sadathshariff) [reply style.css:6]: I'm not using here to keep the structure clean for the landing page

**Review by vivekmehto — APPROVED** [review]
the design looks good and you have written the clean code.

**Review by ImrozNezami — APPROVED** [review]

**Pull request merged**